packages/mobile/src/screens/mood-collection-screen/MoodCollectionScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import Status from 'audius-client/src/common/models/Status'
+import {
+  getCollections,
+  getStatus
+} from 'audius-client/src/common/store/pages/explore/exploreCollections/selectors'
+import { ExploreCollectionsVariant } from 'audius-client/src/common/store/pages/explore/types'
+import { View } from 'react-native'
+
+import { CollectionList } from 'app/components/collection-list'
+import { Header } from 'app/components/header'
+import LoadingSpinner from 'app/components/loading-spinner'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { ExploreCollection as CollectionMetadata } from 'app/screens/explore-screen/collections'
+
+type SmartCollectionScreenProps = {
+  collection: CollectionMetadata
+}
+
+export const MoodCollectionScreen = ({
+  collection
+}: SmartCollectionScreenProps) => {
+  const status = useSelectorWeb(state =>
+    getStatus(state, { variant: ExploreCollectionsVariant.MOOD })
+  )
+  const exploreData = useSelectorWeb(state =>
+    getCollections(state, { variant: ExploreCollectionsVariant.MOOD })
+  )
+
+  return (
+    <View>
```

Comment:
maybe still wrap with <Screen> for consistency?

Reply:
what do you mean?

Reply:
Oh my bad, html preview messed things up, edited